translate/filters/checks.py

```diff
@@ -413,10 +417,19 @@ def run_test(self, test, unit):
         """
         return test(unit)
 
-    def get_ignored_filters(self):
+    def get_checker_name(self):
+        """Extract checker name, for example 'mozilla' from MozillaChecker."""
+        return str(self.__class__.__name__).lower()[:-len("checker")]
```

Comment:
Shouldn't we actually just get each checker to define that name instead of trying to mangle its extraction